src/Session.php

```diff
@@ -416,20 +416,11 @@ public function prepare($yql)
     {
         $statement = new Statement($this, $yql);
 
```

Comment:
Remove cache will dramatically decreasy perforemance, because SDK will go to server for prepare every time.

I suggest save cache of metadata if any of them need to execute query of replace prepare to stub if nothing need for execute query.